### PR TITLE
Implement proper support for data access logs in resource manager modules

### LINKED ISSUE #1384: Data access logs support
Add data access logs support to folders and projects

### PR DESCRIPTION
**Pull request by ludoo**
Fixes #1384

- [x] organization
- [x] folder
- [x] project

### CODE REVIEW

File: modules/folder/README.md [hunk 1]
```diff
@@ -301,6 +303,56 @@ module "folder-sink" {
...
+This example shows how to set a non-authoritative access log configuration:
+
+```hcl
+module "folder" {
+  source = "./fabric/modules/folder"
+  parent = "folders/657104291943"
+  name   = "my-folder"
+  logging_data_access = {
+    allServices = {
+      ADMIN_READ = ["group:organization-admins@example.org"]
```
Comment (juliocc) [inline modules/folder/README.md:319]: I think we should mention somewhere that these are excluded identities. It's not clear at first glance what they mean.

File: modules/folder/variables.tf [hunk 2]
```diff
@@ -89,12 +89,33 @@ variable "iam_additive_members" {
   nullable    = false
 }
 
+variable "iam_policy" {
```
Comment (juliocc) [inline modules/folder/variables.tf:92]: `iam_policy` is mutually exclusive with the other iam variables? If so, I think we should mention it the the description

**Review by juliocc — APPROVED** [review]
Looks great. Left a couple of comments but you can address them later if you want.

File: modules/folder/README.md (hunk 1 above)
Comment (ludoo) [reply modules/folder/README.md:322]: good point, addressed in the most recent commit

File: modules/folder/variables.tf (hunk 2 above)
Comment (ludoo) [reply modules/folder/variables.tf:92]: good point, addressed in the most recent commit

**Pull request merged**